README.md

```diff
@@ -20,7 +20,7 @@ To get a local copy up and running follw these simple example steps:
     - Clone the repo with git clone using the link below
     - git clone hhttps://github.com/ogiogiovictor/bookstore
```

Comment:
- [ ] It would be better if we use `createAsyncThunk` in our project for creating thunk action creators